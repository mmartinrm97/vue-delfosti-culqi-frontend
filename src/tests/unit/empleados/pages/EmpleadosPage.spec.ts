import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import { createApp } from 'vue'
import EmpleadosViewVue from '@/empleados/pages/EmpleadosView.vue'
import { QueryClient, VueQueryPlugin } from '@tanstack/vue-query'

describe('EmpleadosView page', () => {
  it('should render', () => {
    const queryClient = new QueryClient()
    const app = createApp(EmpleadosViewVue)
    app.use(VueQueryPlugin, { queryClient })
    const wrapper = mount(app.component)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
