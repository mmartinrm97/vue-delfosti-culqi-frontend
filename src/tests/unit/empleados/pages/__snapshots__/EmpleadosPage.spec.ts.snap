// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EmpleadosView page > should render 1`] = `""`;
